addons/io_scene_gltf2/io/exp/gltf2_io_user_extensions.py

```diff
@@ -12,11 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def export_user_extensions(hook_name, export_settings, gltf2_object, *args):
-    if gltf2_object.extensions is None:
-        gltf2_object.extensions = {}
+def export_user_extensions(hook_name, export_settings, *args):
+    if args and hasattr(args[0], "extensions"):
+        if args[0].extensions is None:
+            args[0].extensions = {}
 
     for extension in export_settings['gltf_user_extensions']:
         hook = getattr(extension, hook_name, None)
         if hook is not None:
-            hook(gltf2_object, *args, export_settings)
+            try:
+                hook(*args, export_settings)
+            except Exception:
+                pass
```

Comment:
Should the exception get logged to the console?  Might be hard to debug without that.